backend/api/cmd/api/main.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 func main() {
-	app, err := app.FromEnv()
+	app, err := app.FromEnv("api")
```

Comment:
we have the go code back again :(

Reply:
pr on the master branch is updated and here on next branch have to update.